src/config/index.ts

```diff
@@ -97,6 +97,7 @@ export const WHO_IS_MAKING_UPDATE_PAGE = "who-is-making-update";
 export const CONFIRM_OVERSEAS_ENTITY_DETAILS_PAGE = "confirm-overseas-entity-details";
 export const UPDATE_FILING_DATE_PAGE = "update-filing-date";
 export const OVERSEAS_ENTITY_REVIEW_PAGE = "overseas-entity-review";
+export const BENEFICIAL_OWNER_BO_MO_REVIEW_PAGE = "beneficial-owner-bo-mo-review";
```

Comment:
Let's prefix with 'UPDATE' to match the rest of our variables

Reply:
Done.